.github/workflows/release-docker-image.yml

```diff
@@ -68,4 +68,14 @@ jobs:
           docker build -t flight-tracker-consumer .
           docker tag flight-tracker-consumer:latest 058264501037.dkr.ecr.us-east-1.amazonaws.com/flight-tracker-consumer:${BUILD_NUMBER}
           docker push 058264501037.dkr.ecr.us-east-1.amazonaws.com/flight-tracker-consumer:${BUILD_NUMBER}
+          docker logout AWS
+
+      - name: Authenticate to dockerhub
+        run : |
+          docker login -u ewagner2802 --password-stdin $DOCKERHUB_PUSH
```

Comment:
@ewags2 could you rename DOCKERHUB_PUSH to DOCKERHUB_ACCESS_TOKEN or something else that better represents what it is ? 

Also, in order to access Github secrets, you would need to first define an env (preferably at the top of the file). In this case, assuming you rename the secrets, you would have : 

``` yml
env:
  SONAR_TOKEN: ${{secrets.SONAR_TOKEN}}
  SONAR_HOST_URL: ${{secrets.SONAR_HOST_URL}}

  AWS_DEFAULT_REGION: us-east-1
  AWS_ECR_ARN: ${{secrets.AWS_ECR_ARN}}
  AWS_ACCESS_KEY_ID: ${{secrets.AWS_ECR_ACCESS_KEY}}
  AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_ECR_ACCESS_KEY_SECRET}}
  
  DOCKERHUB_ACCESS_TOKEN: ${{secrets.DOCKERHUB_ACCESS_TOKEN}} # line added
```